common.gypi

```diff
@@ -91,8 +93,6 @@
       }],
       ['OS=="mac"', {
         'clang%': 1,
-      }, {
-        'clang%': 0,
```

Comment:
Are these changes (making `clang%` have a `0` default and only changing it on mac) just cosmetic cleanup? If not, could you explain how they are linked to the rest of the commit?

Reply:
It's to make it more explicit that `clang` is always defined, so I moved the else to the top of the file [L25](https://github.com/nodejs/node/pull/23156/files#diff-3c97d28212ec6ec5600be248eb430994R25)
The `clang` variable is assumed to be defined in v8.gyp.